pkg/os/darwin/release_info.go

```diff
@@ -20,8 +21,9 @@ var (
 
 func AtLeast(targetVersion string) (bool, error) {
 	once.Do(func() {
-		macCurrentVersion, errGettingVersion = syscall.Sysctl("kern.osproductversion")
-		logging.Debugf("kern.osproductversion is: %s", macCurrentVersion)
+		macCurrentVersion, _, errGettingVersion = os.RunWithDefaultLocale("sw_vers", "-productVersion")
+		macCurrentVersion = strings.TrimTrailingEOL(macCurrentVersion)
```

Comment:
Is there an EOL to strip?

Reply:
yes.

Reply:
if not, does it harm?

Reply:
if there is no EOL, the person noticing this line is unneeded 3 years from now will have no idea if it was needed at the time, and if removing it will break older systems, or if it was unneeded even at the time.